cdc-accelerator/preview/preview.js

```diff
@@ -328,140 +108,492 @@ const loadScreenSetsMenu = (callback = () => {}) => {
+                })
+                return { screenSetID, screensID }
+            })
+            .filter((screenSet) => screenSet.screensID.length)
+    }
 
-const loadScreenSetCss = async (params) => {
-    // Create and load css file
-    const cssFile = document.createElement('link')
-    cssFile.setAttribute('rel', 'stylesheet')
-    cssFile.setAttribute('type', 'text/css')
-    cssFile.setAttribute('class', LINK_CSS_CLASS)
-    cssFile.setAttribute('href', await getScreenSetCssFilename(params))
+    async loadScreenSetCss(params) {
+        // Create and load css file
+        const cssFile = document.createElement('link')
+        cssFile.setAttribute('rel', 'stylesheet')
+        cssFile.setAttribute('type', 'text/css')
+        cssFile.setAttribute('class', WebScreenSets.#LINK_CSS_CLASS)
+        cssFile.setAttribute('href', await this.getScreenSetCssFilename(params))
 
-    // Remove previous css files
-    Array.from(document.querySelectorAll(`link[rel="stylesheet"].${LINK_CSS_CLASS}`)).forEach((element) => element.remove())
+        // Remove previous css files
+        Array.from(document.querySelectorAll(`link[rel="stylesheet"].${WebScreenSets.#LINK_CSS_CLASS}`)).forEach((element) => element.remove())
 
-    // Load css file
-    document.head.appendChild(cssFile)
-}
+        // Load css file
+        document.head.appendChild(cssFile)
+    }
 
-const getScreenSetCssFilename = async ({ apiKey, screenSetID }) => {
-    const sites = await getConfigSites(ORIGIN)
-    const site = sites.find((site) => site.apiKey === apiKey)
+    async getScreenSetCssFilename({ apiKey, groupID }) {
+        return await Feature.getFeatureFilename(apiKey, `${this.getName()}/${groupID}/${groupID}.css`)
+    }
 
-    let filename = BUILD_DIRECTORY
-    if (site.siteDomain) {
-        filename += `${site.siteDomain}/`
+    async getScreenSetEvents({ apiKey, groupID }) {
+        const filename = await Feature.getFeatureFilename(apiKey, `${this.getName()}/${groupID}/${groupID}.js`)
+        return await this.#getScreenSetEventsFromFile(filename)
     }
-    filename += `webScreenSets/${screenSetID}/${screenSetID}.css`
 
-    return filename
+    async #getScreenSetEventsFromFile(filename) {
+        // Get screenSet JavaScript from build folder
+        let screenJs = await fetch(filename)
+            .then((response) => response.text())
+            .then((data) => data)
+            .catch((error) => {
+                console.log({ error })
+            })
+
+        // Remove comments and \n
+        screenJs = screenJs.replace(/\/\*[\s\S]*?\*\/|([^\\:]|^)\/\/.*$/gm, '')
+        screenJs = '' + screenJs.replaceAll('\n', '')
+
+        let screenSetEvents
+        try {
+            screenSetEvents = eval('(' + screenJs + ')')
+        } catch (error) {
+            alert('Error loading local Screen-Set events from file: \n\n' + filename)
```

Comment:
Replace with `location.reload()`, to enable the build command to finish.

Also find other locations where we are throwing the error to the console and replace them with a reload, ex: webSdk file, emailTemplates metadata

Reply:
Done